src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```
@@ -231,6 +231,10 @@
     <Compile Include="TracingTests\DiagnosticTest.cs" />
     <Compile Include="TracingTests\FakeDiagnosticListenerObserver.cs" />
   </ItemGroup>
+  <!--.Net core 3.0 & .Net standard 2.1 and above-->
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp' AND !$(TargetFramework.StartsWith('netcoreapp2')) AND !$(ReferenceType.Contains('netcoreapp2')) AND !$(ReferenceType.Contains('NetStandard2.0'))">
```

When reference type is .NET Standard, target framework for tests will still be .NET Core.

```suggestion
  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp' AND !$(TargetFramework.StartsWith('netcoreapp2'))">
```